support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.test.tsx

```diff
@@ -96,7 +96,7 @@ describe('Guardian Weekly checkout form', () => {
 					countryId: 'GB',
 					currencyId: 'GBP',
 				},
-				abParticipations: [],
+				abParticipations: {},
```

Comment:
Maybe this is an array as `abParticipations` plural? Don't think it make s a difference in the test to be honest.

Reply:
`abParticipations` is actually an object and not an array, the original mock state was incorrect, it didn't break anything but I've updated it nevertheless.

Reply:
I was looking at this file as the tests did actually flag a regression I would've introduced if they hadn't caught it!